localnet/setup_genesis.sh

```diff
@@ -21,9 +21,10 @@ BUILD_DIR=$(pwd)/localnet/build
 # DATA_DIR=$BUILD_DIR/node4/$CHAIND
 DATA_DIR=$BUILD_DIR/node4/evmosd
 
-GENESIS=$DATA_DIR/config/genesis.json
-TEMP_GENESIS=$DATA_DIR/config/tmp_genesis.json
-CONFIG=$DATA_DIR/config/config.toml
+CONF_DIR=$DATA_DIR/config
+GENESIS=$CONF_DIR/genesis.json
+TEMP_GENESIS=$CONF_DIR/tmp_genesis.json
+CONFIG=$CONF_DIR/config.toml
 
 # create necessary directory for orchestrator node
 mkdir -r "$DATA_DIR"
```

Comment:
`-r` raises the `illegal option` error for the command, did you mean `-p`?
```suggestion
mkdir -p "$DATA_DIR"
```